### PR TITLE
refactor, chore(egen): Removes boilerplate, heading fixes, and other corrections from template

### PR DESCRIPTION
**Pull request by nileshspringml**
**REQUIRED:** 

- Removed boilar code
- Changed REGION variable name to LOCATION
- Hardcode tensorflow version to 2.15.1
- Removed os.getenv(""IS_TESTING"") while cleanup bucket
- Colab enterpise logo added
- Written bullet points into different line. As currently they written on single line
- Different places changed text from first person to second person
- Placed hyperlink at appropriate place where its present with some unwanted things

<br>
Adapted notebook with new template and removed boilerplate code
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb [hunk 1]
```diff
@@ -810,7 +756,7 @@
         "Now define the command-line arguments for your custom training container:\n",
         "\n",
         "- `args`: The command-line arguments to pass to the executable that is set as the entry point into the container.\n",
-        "  - `--model-dir` : For our demonstrations, we use this command-line argument to specify where to store the model artifacts.\n",
+        "  - `--model-dir` : For our demonstrations, use this command-line argument to specify where to store the model artifacts.\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb:759]: "For these demonstrations..." (don't use 1st person plural pronouns)

**Review by kittyabs — COMMENTED** [review]
some suggested edits

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb [hunk 2]
```diff
@@ -571,7 +514,7 @@
         "\n",
         "The files `setup.cfg` and `setup.py` are the instructions for installing the package into the operating environment of the Docker image.\n",
         "\n",
-        "The file `trainer/task.py` is the Python script for executing the custom training job. *Note*, when we referred to it in the worker pool specification, we replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
+        "The file `trainer/task.py` is the Python script for executing the custom training job. *Note*, when you referred to it in the worker pool specification then replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
```
[the remark this answers is not in the document]
Comment (nileshspringml) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Did required changes.

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb [hunk 3]
```diff
@@ -615,7 +558,7 @@
       "source": [
         "#### Task.py contents\n",
         "\n",
-        "In the next cell, you write the contents of the training script task.py. We won't go into detail, it's just there for you to browse. In summary:\n",
+        "In the next cell, you write the contents of the training script task.py. You won't go into detail, it's just there for you to browse. In summary:\n",
```
[the remark this answers is not in the document]
Comment (nileshspringml) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Did required changes.

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb (hunk 1 above)
Comment (nileshspringml) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Did required changes.

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb [hunk 4]
```diff
@@ -395,7 +338,7 @@
         "\n",
         "Otherwise specify `(None, None)` to use a container image to run on a CPU.\n",
         "\n",
-        "Learn more [here](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators) hardware accelerator support for your region\n",
+        "Learn more [hardware accelerator support](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators) for your location\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb:341]: add "about":

- Learn more about [hardware accelerator support](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators) for your location

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb [hunk 5]
```diff
@@ -571,7 +514,7 @@
         "\n",
         "The files `setup.cfg` and `setup.py` are the instructions for installing the package into the operating environment of the Docker image.\n",
         "\n",
-        "The file `trainer/task.py` is the Python script for executing the custom training job. *Note*, when we referred to it in the worker pool specification, we replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
+        "The file `trainer/task.py` is the Python script for executing the custom training job. *Note*, when `trainer/task.py` is referred to it in the worker pool specification, the directory slash is replaced with a dot and the file suffix (`.py`) is dropped (`trainer.task`).\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb:517]: Change "note" to:

- *Note*, when `trainer/task.py` is referred to in the worker pool specification, the directory slash is replaced with a dot and the file suffix (`.py`) is dropped (`trainer.task`).

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb [hunk 6]
```diff
@@ -615,7 +558,7 @@
       "source": [
         "#### Task.py contents\n",
         "\n",
-        "In the next cell, you write the contents of the training script task.py. We won't go into detail, it's just there for you to browse. In summary:\n",
+        "In the next cell, you write the contents of the training script task.py. You don't have to go into detail. It's just there for you to browse. In summary:\n",
```
Comment (kittyabs) [inline notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb:561]: Change to:

- In the next cell, provide the contents of the training script task.py file. You don't have to go into detail. ..."

**Review by kittyabs — COMMENTED** [review]
suggested edits

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb (hunk 4 above)
Comment (nileshspringml) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Did required changes

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb (hunk 5 above)
Comment (nileshspringml) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Did required changes

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb (hunk 6 above)
Comment (nileshspringml) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Did required changes

**Comment by nileshspringml** [comment]
> suggested edits

We did all the required changes.

**Review by katiemn — APPROVED** [review]

**Pull request merged**